compat/src/main/scala-2.13/package.scala

```diff
@@ -0,0 +1,19 @@
+package spire
+package object scalacompat {
+
+import scala.collection.mutable.ArrayBuilder
+import scala.reflect.ClassTag
+
+  def arrayBuilderMake[T]()(implicit tag: ClassTag[T]): ArrayBuilder[T] =
+      ArrayBuilder.make[T]
+
+  def parallelSeq[A](s: Seq[A]): Seq[A] = {
```

Comment:
Add an issue for this too, please. Eventually I'd like to get rid of all parallel collections and replace it by `Future`-based parallelism.